posts/2022/esds-event-prep.md

```diff
@@ -0,0 +1,41 @@
+---
+author: Julia Kent
+date: 2022-11-02
+tags: tutorial hackathon events
+---
+
+# Preparation for a (Re)Introduction to Earth System Data Science (ESDS) Across NCAR/UCAR/UCP
+
+ESDS is hosting a (Re)Introduction to Earth System Data Science (ESDS) Across
+NCAR/UCAR/UCP on **November 10th and 11th** (Thursday-Friday). This event will
+be hybrid, in-person in NCAR's Mesa Lab Main Seminar Room and virtual over Zoom
+or Google Meet. More information is in this [previous post](esds-fall-event.md).
+
+## Environment Install
+
+Please install Miniconda and Jupyter before the ESDS event.
+
+You can follow [Project Pythia instructions to install Python on Jupyter](https://foundations.projectpythia.org/foundations/jupyter.html).
+
+For more information, review the [Jupyter Project Pythia content](https://foundations.projectpythia.org/foundations/getting-started-jupyter.html).
+
+If you have trouble installing miniconda - try to uninstall and reinstall it, making sure you select the correct install location. If you have any aditional questions, feel free to reach out to Julia Kent at jkent@ucar.edu.
+
+Once installed, please create the tutorial environment by downloading the `environment.yml` file from REPOSITORY. Use the terminal command `conda env create -n ENVNAME --file ENV.yml` followed by `conda activate ENVNAME`.
+
+The 08:40 - 09:30: Python Intro and Workspace Set Up slot on Day 1 will be for troubleshooting any problems that we could not resolve beforehand.
+
+## Session homework
+
+The events on Day 1 will follow a flipped classroom model, where you watch some tutorials and do study on your own before coming together in your breakout group to troubleshoot, ask questions, or dive deeper. To get the most out of these sessions, please complete the prep work selected by each session leader.
+
+The breakout groups are:
+
+- [Python Basics](https://docs.google.com/document/d/1ku15eYYGFujpGaiabJCDL4Lo8WBJilG5x2GbjMsjsNU/edit?usp=sharing)
+- [Numpy](https://docs.google.com/document/d/1fCLZaJwQxiV12QHAwVuu5fHv5h6Atoq9xT_rdnEX3OM/edit?usp=sharing)
+- [Visualization]()
```

Comment:
Here's the link to my homework doc

https://docs.google.com/document/d/1pCZNh1NToWETMBwHQr-rj12l-ABJQpdEXK8-IGa6r4E/edit?usp=sharing

Reply:
Great thanks!